test/Conversion/TorchToLinalg/view.mlir

```diff
@@ -0,0 +1,83 @@
+// CHECK:        %[[v1:.*]] = torch_c.to_i64 %[[int1]]
+// CHECK:        %[[int0:.*]] = torch.constant.int 0
+// CHECK:        %[[v2:.*]] = torch_c.to_i64 %[[int0]]
+// CHECK:        %[[c2_i64:.*]] = arith.constant 2 : i64
+// CHECK:        %[[v3:.*]] = arith.addi %[[v2]], %[[c2_i64]] : i64
+// CHECK:        %[[c0_i64:.*]] = arith.constant 0 : i64
+// CHECK:        %[[v4:.*]] = arith.cmpi sge, %[[v2]], %[[c0_i64]] : i64
+// CHECK:        %[[v5:.*]] = arith.select %[[v4]], %[[v2]], %[[v3]] : i64
+// CHECK:        %[[c0_i64_0:.*]] = arith.constant 0 : i64
+// CHECK:        %[[v6:.*]] = arith.cmpi sge, %[[v5]], %[[c0_i64_0]] : i64
+// CHECK:        %[[v7:.*]] = arith.cmpi slt, %[[v5]], %[[c2_i64]] : i64
+// CHECK:        %[[v8:.*]] = arith.index_cast %[[v5]] : i64 to index
+// CHECK:        %[[v9:.*]] = tensor.dim %[[v0]], %[[v8]] : tensor<?x?xf32>
+// CHECK:        %[[v10:.*]] = arith.index_cast %[[v9]] : index to i64
+// CHECK:        %[[v11:.*]] = torch_c.from_i64 %[[v10]]
+// CHECK:        %[[c2_i64_1:.*]] = arith.constant 2 : i64
+// CHECK:        %[[v12:.*]] = arith.addi %[[v1]], %[[c2_i64_1]] : i64
+// CHECK:        %[[c0_i64_2:.*]] = arith.constant 0 : i64
+// CHECK:        %[[v13:.*]] = arith.cmpi sge, %[[v1]], %[[c0_i64_2]] : i64
+// CHECK:        %[[v14:.*]] = arith.select %[[v13]], %[[v1]], %[[v12]] : i64
+// CHECK:        %[[c0_i64_3:.*]] = arith.constant 0 : i64
+// CHECK:        %[[v15:.*]] = arith.cmpi sge, %[[v14]], %[[c0_i64_3]] : i64
+// CHECK:        %[[v16:.*]] = arith.cmpi slt, %[[v14]], %[[c2_i64_1]] : i64
+// CHECK:        %[[v17:.*]] = arith.index_cast %[[v14]] : i64 to index
+// CHECK:        %[[v18:.*]] = tensor.dim %[[v0]], %[[v17]] : tensor<?x?xf32>
+// CHECK:        %[[v19:.*]] = arith.index_cast %[[v18]] : index to i64
+// CHECK:        %[[v20:.*]] = torch_c.from_i64 %[[v19]]
+// CHECK:        %[[v21:.*]] = torch.prim.ListConstruct %[[v11]], %[[v20]] : (!torch.int, !torch.int) -> !torch.list<int>
+// CHECK:        %[[v22:.*]] = torch_c.to_i64 %[[v11]]
+// CHECK:        %[[v23:.*]] = torch_c.to_i64 %[[v20]]
+// CHECK:        %[[c0:.*]] = arith.constant 0 : index
+// CHECK:        %[[v24:.*]] = tensor.dim %[[v0]], %[[c0]] : tensor<?x?xf32>
+// CHECK:        %[[c1:.*]] = arith.constant 1 : index
+// CHECK:        %[[v25:.*]] = tensor.dim %[[v0]], %[[c1]] : tensor<?x?xf32>
+// CHECK:        %[[v26:.*]] = tensor.cast %[[v0]] : tensor<?x?xf32> to tensor<?x?xf32>
+// CHECK:        %[[v27:.*]] = torch_c.from_builtin_tensor %[[v26]] : tensor<?x?xf32> -> !torch.vtensor<[?,?],f32>
+// CHECK:        return %[[v27]] : !torch.vtensor<[?,?],f32>
+
+func.func @torch.aten.view$dynamictest(%arg0: !torch.vtensor<[?,?],f32>) -> !torch.vtensor<[?,?],f32> {
+    %int1 = torch.constant.int 1
+    %int0 = torch.constant.int 0
+    %11 = torch.aten.size.int %arg0, %int0 : !torch.vtensor<[?,?],f32>, !torch.int -> !torch.int
+    %20 = torch.aten.size.int %arg0, %int1 : !torch.vtensor<[?,?],f32>, !torch.int -> !torch.int
```

Comment:
It would be nice to improve these variable names as well (at least to be called %0, %1, ...)